### ISSUE TITLE
line height too small in latex \dfrac

### ISSUE THREAD

**Issue by AdrianDuan** [notable/notable#738]
- **OS Version**: macOS 10.14.5
- **Notable Version**: 1.6.2

### Current behavior
latex \dfrac large fraction render is not correct in Chinese. Line height is too small.

source mark down
```md
* Amdahl定律
  * $T_n = T_0 \div S_n$，其中$T_0$为改进前时间，$T_n$为改进后任务执行时间
  * $F_e=\dfrac{可改进部分占用时间}{改进前总时间}$
  * $S_e=\dfrac{改进前总时间}{改进后总时间}$
  * $S_n=\dfrac{1}{(1-F_e)+\frac{F_e}{S_e}}$
```
render result:
<img width="584" alt="图片" src="https://user-images.githubusercontent.com/26034046/61166848-7a163100-a567-11e9-8003-68cdee7e827d.png">

### Expected behavior
Line height needs to be bigger

**Comment by fabiospampinato** [comment]
There should be more spacing between those expressions, correct?

**Comment by fabiospampinato** [comment]
I believe this is an upstream issue: https://github.com/KaTeX/KaTeX/issues/2031

**Comment by AdrianDuan** [comment]
@fabiospampinato Yup I mean should be more spacing between those expressions. Those expressions are overlapping. Thanks for your help!
